internal/storage/clickhouse.go

```diff
@@ -0,0 +1,91 @@
+package storage
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/ClickHouse/clickhouse-go/v2"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+type ClickHouseConnector struct {
+	cache clickhouse.Conn
```

Comment:
I think it would be clearer if this is named `connection` or `conn`

Reply:
done in https://app.graphite.dev/github/pr/thirdweb-dev/indexer/15/improved-clickhouse-connector-for-readability#